ggml/src/ggml-cuda/norm.cu

```diff
@@ -354,77 +373,206 @@ static void rms_norm_f32_cuda(
+        uint32_t mp_mul_rows, L_mul_rows;
+        init_fastdiv_values(mul_nrows, mp_mul_rows, L_mul_rows);
+        uint32_t mp_mul_channels, L_mul_channels;
+        init_fastdiv_values(mul_nchannels, mp_mul_channels, L_mul_channels);
+        uint32_t mp_mul_samples, L_mul_samples;
+        init_fastdiv_values(mul_nsamples, mp_mul_samples, L_mul_samples);
         if (ncols < 1024) {
-            const dim3 block_dims(WARP_SIZE, 1, 1);
-            rms_norm_f32<WARP_SIZE, true><<<blocks_num, block_dims, 0, stream>>>(x, dst,
-                ncols, stride_row, stride_channel, stride_sample, eps,
-                mul, mul_stride_row, mul_stride_channel, mul_stride_sample,
-                mul_ncols, mul_nrows, mul_nchannels, mul_nsamples);
+            const dim3 block_dims(256, 1, 1);
+            rms_norm_f32<256, true><<<blocks_num, block_dims, 0, stream>>>(x,
+                                                                                 dst,
+                                                                                 ncols,
+                                                                                 stride_row,
+                                                                                 stride_channel,
+                                                                                 stride_sample,
+                                                                                 eps,
+                                                                                 mul,
+                                                                                 mul_stride_row,
+                                                                                 mul_stride_channel,
+                                                                                 mul_stride_sample,
+                                                                                 mul_ncols,
+                                                                                 mul_nrows,
+                                                                                 mul_nchannels,
+                                                                                 mul_nsamples,
+                                                                                 mp_mul_cols,
+                                                                                 L_mul_cols,
+                                                                                 mp_mul_rows,
+                                                                                 L_mul_rows,
+                                                                                 mp_mul_channels,
+                                                                                 L_mul_channels,
+                                                                                 mp_mul_samples,
+                                                                                 L_mul_samples);
```

Comment:
For function calls, please use the previous format with multiple values per lane in order to take up less vertical space.